v19.1/managed-build-a-python-app.md

```diff
@@ -0,0 +1,225 @@
+  	done BOOL NULL,
+  	pub_date TIMESTAMP NULL,
+  	CONSTRAINT "primary" PRIMARY KEY (todo_id ASC),
+  	FAMILY "primary" (todo_id, title, text, done, pub_date)
+    );
+    ~~~
+
+## Step 5: Configure the sample Python app
+
+1. In a new tab, install SQLAlchemy:
+
+    To install SQLAlchemy, as well as a [CockroachDB Python package](https://github.com/cockroachdb/cockroachdb-python) that accounts for some differences between CockroachDB and PostgreSQL, run the following command:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ pip install sqlalchemy cockroachdb
+    ~~~
+
+    For other ways to install SQLAlchemy, see the [official documentation](http://docs.sqlalchemy.org/en/latest/intro.html#installation-guide).
+
+2. Clone the `examples-python` repository to your local machine:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ git clone https://github.com/cockroachdb/examples-python
+    ~~~
+
+3. Navigate to the flask-alchemy folder:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cd examples-python/flask-sqlalchemy
+    ~~~
+
+## Step 6. Generate the application connection string
+
+  1. In the top right corner of the Console, click the **Connect** button.
+
+      The **Connect** modal displays.
+
+        <img src="{{ 'images/v19.1/managed/connect-modal.png' | relative_url }}" alt="Connect to cluster" style="border:1px solid #eee;max-width:100%" />
+
+  2. From the **User** dropdown, select `maxroach`.
+  3. Select a **Region** to connect to.
+  4. From the **Database** dropdown, select `todos`.
+  5. On the **Connect Your App** tab, click **Copy connection string**.
+
+      You will need to replace the `<password>` and `<certs_dir>` placeholders with your SQL username's password and the absolute path to your `certs` directory, respectively.
+
+## Step 7. Connect the application to the database
+
+  1. Go back to the `flask-alchemy` folder.
+  2. In the `hello.cfg` file, replace the value for the `SQLALCHEMY_DATABASE_URI` with the application connection string you generated in Step 6.
+
+  {% include copy-clipboard.html %}
+  ~~~
+  SQLALCHEMY_DATABASE_URI = 'cockroachdb://maxroach:Q7gc8rEdS@<region>.<cluster_name>:26257/todos?sslmode=verify-full&sslrootcert=<absolute path to CA certificate>'
+  ~~~
+
+  {{site.data.alerts.callout_info}}
+  You must use the `cockroachdb://` prefix in the URL passed to [`sqlalchemy.create_engine`](https://docs.sqlalchemy.org/en/latest/core/engines.html?highlight=create_engine#sqlalchemy.create_engine) to make sure the [`cockroachdb`](https://github.com/cockroachdb/cockroachdb-python") dialect is used. Using the `postgres://` URL prefix to connect to your CockroachDB cluster will not work.
+  {{site.data.alerts.end}}
+
+## Step 8. Test the application
+
+  1. Run the `hello.py` code:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ python hello.py
+    ~~~
+
+    The application should run at [http://localhost:80](http://localhost:80)
```

Comment:
I think it runs on `localhost:5000` by default, not `localhost:80` (`:80` is only allowed if the application is running as `root`, which is not recommended).